### ISSUE TITLE
"refrigerated" vs "unrefrigerated"; "not preserved" terms

### ISSUE THREAD

**Issue by ghost** [FoodOntology/foodon#49]
Currently FoodOn has "food chilling process" but no explicit class for "food (refrigerated)" , being an output of that process applied to food material.  Likewise there is no "food (unrefrigerated)".  Both have to be taken in the strongest, timeless sense that refrigerated = a food was at some point output of a food chilling process; unrefrigerated = a food was never at any point output of an (anthropocentric) food chilling process.

Also FoodOn does not have a "not preserved" term, equivalent to the stronger claim that a food was never the output of some preservation process.

**Comment by mcourtot** [comment]
do you have something for food (frozen)? or do you distinguish at the level of the process, e.g., "food chilling process" vs "food freezing process"

**Comment by Public-Health-Bioinformatics** [comment]
Yes, we've got "'food (frozen)': output of some 'food freezing process' "  . All food processes have input 'food material'

Currently freezing and chilling are kept separate as the condition of being output from a "chilled" process (-1°C and 7°C) is different from the output conditions of a complete freezing process (I see the definitions adapted from LanguaL require some improvement though). I see your point that chilling is a prelude to freezing; the external conditions may be the same. Ideally we remove remove intentionality from the definitions. Looks like we have to get PATO to add a new subclass of energy, 'thermal energy, and a new 'decreasing thermal energy' quality over time to represent the thermal loss, and then criteria on end state. I suppose that crossed your mind? Cooling and freezing processes would then have output equilibrium states with temperature qualifiers.

Currently:
Chilling: "Cooling and keeping at a temperature between 30°F and 45°F (-1°C and 7°C). Regarding food preservation regulations, the temperature range depends on national legislation."
Freezing: "preservation by freezing a food product and keeping it at a temperature below the freezing point (20-30 degrees) without regard to the product's physical state."

**Comment by ddooley** [comment]
Anders Møller of Danish Food Informatics (LanguaL lead curator) has conveyed to me a lot more information on the subject which will be useful in defining freezing and chilling. I asked about the current LanguaL definitions above. He says: 

Those rules are old, but valid, world-wide physical/hygienic rules (“rules of thumb”) – nothing new there.
 
In general, there are two empirical fix points 40 deg. F/4.4 deg. C (“chill”) and 0 deg. F/-17.8 deg. C (“freezing”). That is where we have the recommended temperatures for the refrigerator (40 deg. F) and the home freezer (0 deg. F) from – nice round numbers.
 
Chilling: keeping the food cooled/refrigerated below the “danger zone” (bacterial and fungi growth) – usually for a “limited” (depending on product and its properties) amount of time – without the formation of ice crystals in the product. For chill foods it must be stated on the label (national/regional legislation/regulation) at (or below) which temperature the food product should be stored. E.g. the current CFR 21 still says “Maintaining refrigerated foods at 45 deg. F (7.2 deg. C) or below as appropriate for the particular food involved”. In Denmark pasteurized milk can be stored at a temperature “below 8 deg. C, best below 5 deg. C”.
Furthermore, every country/region has specific regulations on max. temperatures allowed for specific chill food products and the max. temperatures are dependent of the food/food product and how perishable it is, salinity/water activity, acidity, etc.
 
I found this popular “chilling” illustration from a manufacturer of refrigerators:
 
![image001](https://user-images.githubusercontent.com/4000582/46235398-0041ed80-c32f-11e8-8688-af030ac11c21.png)

Freezing: in general, it just means that the product has a temperature below the freezing point, i.e. all water in the product has solidified and bacterial growth is slowed down considerably (or almost stopped). Other decaying processes, like oxidation processes, are slowed down considerably - rule of thumb: for every 10 degrees (C) lowering of the temperature, the reaction rates are halved (and vice versa).
The freezing point depends on the dissolved compounds in the foods and is usually below the 32 deg. F/0 deg. C – the so-called freezing point depression. Hence the range “20-30 degrees F”.
 
Then a completely different issue is the actual processes involved in bringing the foods to the chilled stage or the frozen stage – but that is a completely different story J
 
Please be aware that there might be slightly different definitions of the foods in this ”group” depending on where you are in the world, e.g. “chilled food” has a distinct meaning of being a manufactured/pre-packed food (stored below 8 deg. C) in some “environments”, like in Britain (https://www.chilledfood.org/the-history-of-uk-chilled-foods/).  The Wikipedia definition (https://en.wikipedia.org/wiki/Chilled_food) is different and not completely “right” in the temperature range. And so on – the reason being that there is no fixed or right overall temperature definition – except (maybe) in the national(/regional) food regulations – and it is dependent of the food product/geographical location.
 
You may want to look at some links with examples for further information:
 
http://www.foodserviceeurope.org/gallery/59/European%20Guide%20to%20Good%20Practise%20for%20Food%20Hygiene%20in%20the%20Contract%20Catering%20Sector%20(May%202009).pdf
http://www.health.state.mn.us/foodsafety/cook/temperature.html
https://www.chilledfood.org/temperature/
https://www.newfoodmagazine.com/article/573/2c-to-12c-not-chilled-but-not-frozen/
https://www.choice.com.au/home-and-living/kitchen/fridges/articles/temperature-guide
https://www.foodsafety.gov/keep/basics/chill/index.html
https://www.fda.gov/food/resourcesforyou/consumers/ucm253954.htm
https://www.foodstandards.gov.scot/consumers/food-safety/at-home/storing-food
 
and this is the best J https://blog.liebherr.com/appliances/my/ideal-freezer-temperature/ - although about (deep-) frozen food, it gives you a good impression of how the good round numbers 40 and 0 (deg. F) were derived J

... thanks Anders!

**Comment by mateolan** [comment]
I think the fundamental design is still incorrect here. We need to base these vocabularies on science, in this case thermodynamics, specifically enthalpy. Secondarily, we can derive terms for use in the regulatory or consumer vernaculars depending the specific time and energy thresholds achieved by a specific processing step. Refactoring FoodOn to use a separate processing ontology that can stand on its own, will go a long way toward these efforts. This will also enable real computational food science in terms of associating and predicting nutritional and flavor outcomes associated with specific ingredients undergoing specific processes. It's just plain silly to talk about how long something was chilled of you don't know the starting temperature, the mass, and the pressure. These are basic heat and mass transfer variables that every food scientist learns. Also, FWIW we need to have a similar conversation about dispersions: foams, emulsions, aerosols, etc. Let's start with science.

**Comment by ddooley** [comment]
I agree - I was trying to address the fundamental science point when I said above "Looks like we have to get PATO to add a new subclass of energy, 'thermal energy, and a new 'decreasing thermal energy' quality over time to represent the thermal loss, and then criteria on end state. I suppose that crossed your mind? Cooling and freezing processes would then have output equilibrium states with temperature qualifiers".  Then we can refactor regulatory / conventional subclasses off of that.  Question for me is how much to get PATO to take on since it is closest to general-purpose material ontology, vs. some more specific physics one?
